Client/src/Pages/NotFound/index.jsx

```diff
@@ -1,10 +1,59 @@
-import React from 'react'
-import './index.css'
+import Button from "../../Components/Button";
+import "./index.css";
+import React from "react";
+import PropTypes from "prop-types";
+import { Typography } from "@mui/material";
 
-const NotFound = () => {
-    return (
-        <div>NotFoundPage</div>
-    )
-}
+/**
+ * Support for defaultProps will be removed from function components in a future major release
+ * So That why we're using JavaScript default parameters instead.
+ */
+const DefaultValue = {
+  title: "We cannot find this page",
+  desc: "Either the URL doesn’t exist, or you don’t have access to it.",
+};
 
-export default NotFound
\ No newline at end of file
+/**
+ * Renders an error page component with a customizable title and description.
+ *
+ * @component
+ * @example
+ * // Usage:
+ * <ErrorPage
+ *   title="Page Not Found"
+ *   desc="The requested page could not be found."
+ * />
+ *
+ * @param {Object} props - The component props.
+ * @param {string} [props.title="We cannot find this page"] - The title of the error page.
+ * @param {string} [props.desc="Either the URL doesn’t exist, or you don’t have access to it."] - The description of the error page.
+ * @returns {JSX.Element} The rendered error page component.
+ */
+const NotFound = ({ title = DefaultValue.title, desc = DefaultValue.desc }) => {
+  return (
+    <div className="not-found-page">
+      <div className="not-found-template">
```

Comment:
It's looking better! Can you use [Stack](https://mui.com/material-ui/react-stack/) here? 

It's desinged to arrange components vertically and horizontally so it's got this exact use case in mind.  It would be great to minimize the custom styles here

Reply:
Fixed ✅